docs/pages/cli/faucet.mdx

```diff
@@ -1,3 +1,5 @@
 # mud faucet
 
-**Coming soon**
+This command line activates the faucet for the obsolete testnet.
```

Comment:
```suggestion
This command is deprecated. It can still be used to interact with the deprecated faucet service on the deprecated "lattice testnet", but will be removed in the future.
```